api/resources_portal/views/organization.py

```diff
@@ -21,13 +21,16 @@ class OrganizationDetailSerializer(OrganizationSerializer):
     owner = UserRelationSerializer()
     members = UserRelationSerializer(many=True)
 
-    def update_members(self, organization, members):
+    def update_members(self, organization, owner_id, members):
         if not members:
```

Comment:
if there are no members we would still want to make sure that the owner is in there on create?

Reply:
awesome!